src/syscheckd/db/tests/fimDBTests/fimDBImpTests.cpp

```diff
@@ -238,4 +236,27 @@ TEST_F(FimDBFixture, executeQueryFailException)
     fimDBMock.executeQuery(itemJson, callback);
 }
 
+TEST_F(FimDBFixture, fimSyncPushMsgSuccess)
+{
+    const std::string& data = "testing msg";
+    auto rawData{data};
+    const auto buff{reinterpret_cast<const uint8_t*>(rawData.c_str())};
+
+    EXPECT_CALL(*mockRSync, pushMessage(std::vector<uint8_t> {buff, buff + rawData.size()}));
+    EXPECT_CALL(*mockLog, loggingFunction(LOG_DEBUG_VERBOSE, "Message pushed: " + data));
+
+    fimDBMock.fimSyncPushMsg(data);
+}
+
+TEST_F(FimDBFixture, fimSyncPushMsgException)
+{
+    const std::string& data = "testing msg";
```

Comment:
This could be write like `const std::string("testing msg")`

Reply:
done